code/modules/vore/eating/bellymodes_vr.dm

```diff
@@ -85,9 +85,9 @@
 				var/mob/living/silicon/robot/R = owner
 				R.cell.charge += 20*(digest_brute+digest_burn)
 			if(offset) // If any different than default weight, multiply the % of offset.
-				owner.nutrition += offset*(2*(digest_brute+digest_burn)/difference) // 9.5 nutrition per digestion tick if they're 130 pounds and it's same size. 10.2 per digestion tick if they're 140 and it's same size. Etc etc.
+				owner.nutrition += offset*(3*(digest_brute+digest_burn)/difference) // 9.5 nutrition per digestion tick if they're 130 pounds and it's same size. 10.2 per digestion tick if they're 140 and it's same size. Etc etc.
```

Comment:
I suppose the comment on the end about nutrition per tick is no longer accurate (but has it ever been :thinking:)?

Reply:
Yeah hasn't really been since the customizable damage values.